packages/vscode/src/plugins/nextjs/addJSDoc.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  ArrowFunction,
+  FunctionDeclaration,
+  SourceFile,
+  TypeGuards,
```

Comment:
TypeGuards is deprecated, can you use something else here?

Reply:
I tried not all of the nodes have this functionality yet